### PR TITLE
[hr_holidays_from_ics]: load Calendars from URL of the iCal file

### PR DESCRIPTION
**Pull request by andreinl**
This module provides functionality to load Calendars from URL of the iCal file (*.ics)

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Shouldn't it be `hr_holidays_public_from_ics`?

**Comment by andreinl** [comment]
> Shouldn't it be `hr_holidays_public_from_ics`?

Personally I don't understand the use of word "public" here.
Though as an extension of hr_holidays_public module the name `hr_holidays_public_from_ics` can be more appropriate. 
If you want I can rename it. Just let me know

File: hr_holidays_from_ics/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+{
+    'name': "HR Holidays from ics",
+    'version': '12.0.0.0.0',
+    'license': 'AGPL-3',
+    'category': 'Human Resources',
+    'author': "Andrei Levin, "
+              "Odoo Community Association (OCA)",
+    'summary': """
+        Load holidays from URL that points to iCal file (*.ics)""",
+    'website': "http://www.didotech.com",
```
Comment (nikul-serpentcs) [inline hr_holidays_from_ics/__manifest__.py]: https://github.com/OCA/hr

File: hr_holidays_from_ics/models/hr_holidays.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+try:
+    import vobject
+except (ImportError, IOError) as err:
+    _logger.debug(err)
+
+
+class HrHolidaysPublic(models.Model):
+    _inherit = 'hr.holidays.public'
+
+    ics_url = fields.Char(string="ICS Url", required=False)
```
Comment (nikul-serpentcs) [inline hr_holidays_from_ics/models/hr_holidays.py]: @andreinl No need to `required=False`

File: hr_holidays_from_ics/views/hr_holidays_view.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data>
```
Comment (nikul-serpentcs) [inline hr_holidays_from_ics/views/hr_holidays_view.xml]: IMO remove `<data>`

**Review by nikul-serpentcs — COMMENTED** [review]
netpicking

File: hr_holidays_from_ics/__manifest__.py (hunk 1 above)
Comment (andreinl) [reply hr_holidays_from_ics/__manifest__.py]: I don't understand this

File: hr_holidays_from_ics/models/hr_holidays.py (hunk 2 above)
Comment (andreinl) [reply hr_holidays_from_ics/models/hr_holidays.py]: explicit is better than implicit

File: hr_holidays_from_ics/views/hr_holidays_view.xml (hunk 3 above)
Comment (andreinl) [reply hr_holidays_from_ics/views/hr_holidays_view.xml]: What's wrong with data?

Comment (nikul-serpentcs) [reply hr_holidays_from_ics/views/hr_holidays_view.xml]: No need to `<data>`, In xml file

File: hr_holidays_from_ics/__manifest__.py (hunk 1 above)
Comment (nikul-serpentcs) [reply hr_holidays_from_ics/__manifest__.py]: website URL should be `https://github.com/OCA/hr`

https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10

Comment (andreinl) [reply hr_holidays_from_ics/__manifest__.py]: Ok

File: hr_holidays_from_ics/views/hr_holidays_view.xml (hunk 3 above)
Comment (andreinl) [reply hr_holidays_from_ics/views/hr_holidays_view.xml]: Ok

**Comment by andreinl** [comment]
I renamed a module in hr_holidays_public_from_ics as requested by Pedro Baeza and applied changes recommended by Nikul Chaudhary. Do I need to create a new Pull request and close this one?

**Comment by jarroyomorales** [comment]
Noo, there is no need for creating a new PR, its is good like this. :wink:

**Pull request closed without merging**